doc/xrcvd/transceiver-monitor-hld.md

```diff
@@ -90,7 +90,8 @@ New Transceiver info table and transceiver DOM sensor table will be added to sta
 	; Defines Transceiver Status info for a port
 	key                          = TRANSCEIVER_STATUS|ifname     ; Error information for SFP on port
 	; field                      = value
-	status                       = 1*255VCHAR                    ; code of the SFP status (plug in, plug out or error)
+	status                       = 1*255VCHAR                    ; code of the SFP status (plug in, plug out)
+    error                        = 1*255VCHAR                    ; SFP error (N/A or a string like error1 | error2 )

```

Comment:
Looks like misalignment due to a mix of tabs and spaces. Can you please replace tabs with spaces?

Reply:
updated, thanks.